README.md

```diff
@@ -580,21 +580,60 @@ Name | Type | Required | Description | Example
 --- | --- | --- | --- | ---
 remotePath | string | yes | The full path to the remote file or a file inside an application bundle | `/sdcard/myfile.txt` or `@my.app.id/path/in/bundle`
 
+### mobile: startActivity
```

Comment:
What about adding a link to this section in `Start app activity (POST /appium/device/start_activity)` in https://github.com/appium/appium-uiautomator2-driver#applications-management ?

Reply:
Added an anchor